### DISCUSSION TITLE
Solar anatomy

### DISCUSSION THREAD

**Discussion by YhonLimon** [google/model-viewer#3052] [Show and tell]
My name si Jhon Silva Alé, i'm from Chile. 
This year i designed some resources with model-viewer (to webAR). I want to share them, I am very proud of them. 
This is a model of the Sun's anatomy in 3D: 

https://universopantagina.cl/anatomiadelsol  

The text, button and audio are in Spanish. It is a spanish easy to understand.

![2](https://user-images.githubusercontent.com/36046618/145869172-df18605d-91eb-4b9e-9754-3171e494e71d.jpg)

Thanks!

**Comment by elalish** [comment]
That's very cool, thanks for  sharing! Google translate actually did a pretty decent job on the website too. It does make me wonder though: What is  bright enough to make  the sun cast a shadow? 😁
